src/less/components/payex/radio.less

```
@@ -6,9 +6,117 @@
 
 	input[type="radio"] {
 		+ label {
+
 			&:before {
 				border: 2px solid @radiobutton-border;
 			}
 		}
+
+		&:is(:hover, :focus-visible) + label:before {
+			box-shadow: 0 0 0 0.375rem var(--gray);
+		}
+
+		&:checked + label:before {
+			background-position: center;
+			background-image: url("../../../icons/payex/circlepx.svg");
+		}
+	}
+}
+
+.radio input[type=radio][disabled]:checked+label:before {
+	background-image: url("../../../icons/payex/circlepx_disabled.svg");
+}
+
+.checkmark {
+	
+    label {
+        border: 1px solid var(--gray);
+        outline: none !important;
```

why removing outline to replace it by borders ?
If the change is motivated by another reason than the change of color please let meknow, I'm very open to revisit how we style this component, especially if it's because you've spotted some accessibility issues or something like this.

If the change is only to achieve a change of color then I think you can achieve if more easily by only modifying the outline color:

```suggestion
        outline-color: var(--gray);
```

this goes for teh rest of the file, everywhere `outline` is changed for `border`